TYPO3.Fluid/Classes/TYPO3/Fluid/ViewHelpers/Validation/IfHasErrorsViewHelper.php

```diff
@@ -52,18 +53,29 @@ class IfHasErrorsViewHelper extends AbstractConditionViewHelper
      */
     public function render($for = null)
     {
+        return $this->renderInternal();
```

Comment:
Can we maybe move this render "implementation" to the abstract class, so this single line method is not necessary in every derived class? Or is required to have a properly annotated render() function with all parameters (for documentation?)?